.editorconfig

```diff
@@ -1,9 +0,0 @@
-root = true
```

Comment:
Are we providing an alternative way to set code style?

Reply:
Honestly, not sure - what would your preference be here?

Reply:
I think we need it in, but might need a tweak so it doesn't force the UI code to follow the same code style